api/src/main/java/org/pmiops/workbench/db/dao/ParticipantCohortStatusDaoImpl.java

```diff
@@ -75,6 +123,82 @@ public void saveParticipantCohortStatusesCustom(List<ParticipantCohortStatus> pa
         }
     }
 
+    @Override
+    public List<ParticipantCohortStatus> findAll(Long cohortReviewId, List<String> filtersList, PageRequest pageRequest) {
```

Comment:
This is maybe a nit, but I would propose passing in List<Filter> here instead of List<String>; let the controller be responsible for deserialization.